docs/.vitepress/theme/style.css

```
@@ -0,0 +1,148 @@
+  max-width: 100%;
+  height: auto;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
```

Biome (2.4.15)

[error] This `@import` is in the wrong position.

(lint/correctness/noInvalidPositionAtImportRule)